Description: service: use SIGINT to stop openconnect on disconnect
 OpenConnect exits immediately on SIGTERM, but does a controlled shutdown on
 SIGINT.
 https://bugzilla.gnome.org/show_bug.cgi?id=775165
Author: Mike Miller <mtmiller@debian.org>
Origin: upstream, https://gitlab.gnome.org/GNOME/NetworkManager-openconnect/commit/a27fd1a5e9ad66ca1f45fda771c8e20d2292aee2
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=775165
Bug-Debian: https://bugs.debian.org/820892
Last-Update: 2019-05-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/nm-openconnect-service.c
+++ b/src/nm-openconnect-service.c
@@ -603,7 +603,7 @@
 	NMOpenconnectPluginPrivate *priv = NM_OPENCONNECT_PLUGIN_GET_PRIVATE (plugin);
 
 	if (priv->pid) {
-		if (kill (priv->pid, SIGTERM) == 0)
+		if (kill (priv->pid, SIGINT) == 0)
 			g_timeout_add (2000, ensure_killed, GINT_TO_POINTER (priv->pid));
 		else
 			kill (priv->pid, SIGKILL);
